server/sms/nexmo_sms_sender.go

```diff
@@ -231,7 +231,7 @@ func (ns *NexmoSender) sendSms(sms *NexmoSms) (*NexmoMessageResponse, error) {
 		pNexmoResponseInternalErrors.Inc()
 		return nil, ErrSMSResponseDecodingFailed
 	}
-	logger.WithField("messageResponse", messageResponse).Info("Actual nexmo response")
+	logger.WithField("messageResponse", messageResponse).WithField("order_id", sms.ClientRef).Info("Actual nexmo response")
```

Comment:
here and in other places in the PR:
the field name previously used in our services for order-id was: csOrderId
it might be easier to search in an uniform manner